### PR TITLE
Added support for html content inside chat message. (#2134)

### PR DESCRIPTION
**Pull request by pablodiazv**
### Please read and mark the following check list before creating a pull request:

 - [X] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [X] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:
This resolve an small feature requested by #2134 
Adds support for a new type of chat message: **html** that allows more options to automatic chatbots to include links, buttons and other html content.

### CODE REVIEW

**Comment by crates** [comment]
@pablodiazv / @akveo-private:

What's stopping this feature from getting merged? It's been almost a year since the initial pull request, and we REALLY need the ability to put links into messages.

**Comment by crates** [comment]
@akveo / @nnixaa / @mishkolesnikov / @Tibing / @yggg / @piotrbrzuska / @SashaSkywalker / @sashaqred / @elupanov:
Sorry to tag so many people from your team, but we really need movement on this item right away.
This is an important feature for us immediately. It looks like @pablodiazv solved it a year ago.
Is there any chance your team might be willing to quickly get this code merged to the trunk?

File: src/framework/theme/components/chat/chat-message-html.component.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  @Input() date: Date;
+
+  /**
+   * Message send date format, default 'shortTime'
+   * @type {string}
+   */
+  @Input() dateFormat: string = 'shortTime';
+
+  public get htmlMessage(): SafeHtml {
+    return this.sanetize.bypassSecurityTrustHtml(this.message);
```
Comment (andreipadolin) [inline src/framework/theme/components/chat/chat-message-html.component.ts:51]: It would be nice to move `sanetize` into pipe.

Comment (elupanov) [reply src/framework/theme/components/chat/chat-message-html.component.ts:51]: @pablodiazv we could merge the PR after this enhancement

Comment (andreipadolin) [reply src/framework/theme/components/chat/chat-message-html.component.ts:51]: Hi @crates, first of all much appreciate your efforts in contributing. We a releasing a new feature that allow to use custom template messages including links and other html elements within next release. Please feel free to close your PR after that feature released.

**Comment by yggg** [comment]
Closed in #2750

**Pull request closed without merging**